Codebat 1 post by Zach Williams

zbay opened the pull request:
Hello, @eah13. I've finished the first round of Codebat exercises. Here's the link on my personal site: http://zbay.github.io/spring2014/2014/02/05/zbayCodebat1.html

You may have noticed that this is the second pull request I've made for this assignment. I realized a major formatting mistake after the pull request and couldn't figure out how to commit my edits to it. Therefore, I deleted my old post, made a duplicate one, and closed the old pull request before generating this second request.

zbay committed:
Merge pull request #2 from silshack/gh-pages

Merging recent changes into my personal site

zbay committed:
Created my codebat post

zbay committed:
Updated my Codebat post

Got the code to format correctly and added the link to my screenshots.

zbay committed:
Fixed the link

Got the markdown format correct for adding a link.

zbay committed:
Renamed my file to the proper date

zbay committed:
Added a line

zbay committed:
Fixed my number issue and code display

zbay committed:
Deleted a superfluous line

zbay committed:
Delete 2014-02-05-zbayCodebat1.md

Need to submit a new pull request since I can't update my current one.

zbay committed:
Respawned my Codebat post

Going to make a new pull request!

eah13 commented:
@zbay RE pull requsts: your commits should have been sowing up on the old pull request before this one was opened.  Sometimes you have to do a hard reload in the browser (F5 or Cmd-shft-R depending on your OS) to force it to get another copy of the page.  Nice workaround anyways!

eah13 commented on _posts/2014-02-05-zbayCodebat1.md:
```diff
@@ -0,0 +1,49 @@
+date: 2014-02-05
+---
+
+Finished the first bat of Codebat problems. [Here are my screenshots as proof.](http://imgur.com/a/3vNEv#0) Note that they are in ALBUM form on imgur.
```
Please embed your screenshots in your post as we did before.

eah13 commented on _posts/2014-02-05-zbayCodebat1.md:
````diff
@@ -0,0 +1,49 @@
+  return "off"
+```
+  
+Number 2. Logic-1 > caught_speeding. I had some absurdly complicated if statements until I realized that since 'is_birthday' gives you 5 mph leniency in all circumstances, that I could just subtract 5 from 'speed' and base all of my tests on 'speed'. That way, none of my returning statements needed to test is_birthday.
````
Bingo!  Nice job working through this one.

eah13 commented:
Nice work, especially on the narratives.  Embed those screenshots and you're ready to go.

zbay committed:
Embeds+indents

Embedded my images as apparently required. Also realized that my code sections weren't indented properly.

zbay commented:
I embedded the images and fixed some indentation problems.

eah13 commented:
:boom:
